sys/net/gnrc/application_layer/static/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE := gnrc_static
```

Comment:
I really don't like the name. `gnrc_static` sounds like you do something static to GNRC. However, all this is is a static address initialization for the border router's wireless interface. Can we find a name that reflects that? `gnrc_ipv6_br_static_addr` e.g.

Reply:
Let's go with `gnrc_ipv6_static_addr` since this is not limited to border routers, but works with any (non-border) router configuration.